Trappist/src/Promact.Trappist.DomainModel/ApplicationClasses/Question/QuestionDetailAC.cs

```diff
@@ -1,15 +1,20 @@
 ﻿using Promact.Trappist.DomainModel.Enum;
+using System.ComponentModel.DataAnnotations;
 
 namespace Promact.Trappist.DomainModel.ApplicationClasses.Question
 {
     public class QuestionDetailAC
     {
+        [Required]
         public string QuestionDetail { get; set; }
 
+        [Required]
         public QuestionType QuestionType { get; set; }
 
+        [Required]
         public DifficultyLevel DifficultyLevel { get; set; }
 
+        [Required]
```

Comment:
No need to make int as required. it will be by default initialized with 0 value.